lib/authorized_persona/authorization.rb

```diff
@@ -26,7 +24,14 @@ def authorize_persona(class_name:, current_user_method: nil) # rubocop:disable M
           raise AuthorizedPersona::Error, "#{class_name} must be an AuthorizedPersona::Persona"
         end
 
-        self.authorization_current_user_method = current_user_method || :"current_#{authorization_persona.model_name.singular_route_key}"
+        model_name = if authorization_persona.respond_to?(:model_name)
+                       authorization_persona.model_name.singular_route_key
+                     else
+                       authorization_persona.name.underscore
```

Comment:
i think `underscore` comes from ActiveSupport, but since you're including railties you're basically saying "this project is a rails project"

Reply:
Yeah, railties depends on it, but not activemodel. I require active support explicitly in authorized_persona.rb.